chapter_5/conan_conanfile_example/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+# CMakeLists creating a simple executable that links to a library retrieved with conan from a conanfile.txt
+#
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required(VERSION 3.21)
+
+project(
+    conan_conanfile_example
+    VERSION 1.0
+    DESCRIPTION
+        "A simple C++ project to demonstrate creating a standalone executables"
+    LANGUAGES CXX
+)
+find_program(CONAN conan)
+
+# if built within conan use the file from local cache
+if(CONAN)
+  
+  # Download conan.cmake if it does not already exist
+  if(NOT EXISTS "${CMAKE_CURRENT_BINARY_DIR}/conan.cmake")
+    message(STATUS "Downloading conan.cmake from https://github.com/bernedom/cmake-conan")
```

Comment:
GitHub repository in message is different than the actual source

Reply:
Copy paste is dangerous 😉